mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = async (req, res) => {
```

Comment:
The function name `deleteOrder` should be consistent with the other function names in the file, which are in camelCase. It should be renamed to `deleteOrderById`.